dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblem.java

```diff
@@ -28,15 +30,18 @@
   private AccessModifier modifier;
 
   InaccessibleClassProblem(
-      ClassFile sourceClass, ClassFile targetClass, Symbol classSymbol, AccessModifier modifier) {
+      ClassFile sourceClass,
+      @Nullable ClassFile targetClass,
+      ClassSymbol classSymbol,
+      AccessModifier modifier) {
     super("is not accessible", sourceClass, classSymbol, targetClass);
     this.modifier = modifier;
   }
 
   @Override
   public final String toString() {
     StringBuilder message = new StringBuilder();
-    message.append("Class " + getTargetClass().getBinaryName());
```

Comment:
`getTargetClass()` may return `null`. It caused the NullPointerException in the issue.